android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -133,6 +133,7 @@ public void configure(NativeViewGestureHandler handler, ReadableMap config) {
     @Override
     public void extractEventData(NativeViewGestureHandler handler, WritableMap eventData) {
       eventData.putBoolean("pointerInside", handler.isWithinBounds());
+      eventData.putDouble("numberOfTouches", handler.getNumberOfPointers());
```

Comment:
OMG. I don't know how this slipped into here under this name. Do you mind renaming to `numberOfPointers` here and on iOS for consistency?

Reply:
Renaming is not a good idea since some people are already using this. I just borrowed the name that is used on iOS 🤔 If you really want to change it, it's better to set a getter for `numberOfTouches` in `nativeEvent` that returns `numberOfPointers` and warns that it's deprecated.

Reply:
@tsapeta, I do not consider it as good motivation. Well, I don't we need to care about field which are not documented and appears on only one platform

Reply:
Fields returned in the event are not documented at all, so certainly people are checking what it has and trying on their own. If someone has reported a problem, it means that someone is using it.
Also, we don't have a changelog where we could explain that change.

Reply:
the lib is still in alpha so breaking changes is just sth people will have to deal with when upgrading. Let's make sure to communicate this change to people reporting the issue and also when the next release comes out. We don't have capacity at the moment to support backward compatibility.

Also, we now publish change logs in the releases section on GH

Reply:
Also, we have event field definition in typescript def file, so lets make sure to update it there too